apps/pie-docs/src/components/select-input/overview/overview.md

```diff
@@ -0,0 +1,400 @@
+    ]
+} %}
+
+---
+
+## Variations
+
+### Default
+
+{% contentPageImage {
+    src:"../../../assets/img/components/select-input/variations-default.svg",
+    alt: "Default select input component showing a dropdown with options.",
+    width: 320
+} %}
+
+### Grouped options
+
+Organizing options into logical categories can help users better understand the content within the select input.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/select-input/variations-grouped-options.svg",
+    alt: "Select input with options organized into logical groups for better categorization.",
+    width: 544
+} %}
+
+---
+
+## Modifiers
+
+### Leading icon
+
+Use a leading icon for non-interactive icons that visually support the select input.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/select-input/modifiers-leading-icon.svg",
+    alt: "Select input with a leading icon that provides visual context.",
+    width: 256
+} %}
+
+---
+
+## Sizes
+
+### Height
+
+Use a consistent select input height when it is being used alongside other form components on the same page.
+
+{% contentLayout %}
+  {% contentItem %}
+    <h3>Small</h3>
+    <p>Use when there isn't enough vertical space for the default select input size.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/select-input/sizes-height-small.svg",
+      width: 256,
+      alt: "Small-sized select input for limited vertical space."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    <h3>Medium</h3>
+    <p>Default size and should be used whenever possible.</p>
+    {% contentPageImage {
+      src: "../../../assets/img/components/select-input/sizes-height-medium.svg",
+      width: 256,
+      alt: "Medium-sized select input."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
+
+### Large
+
+Choose this size when there is a lot of space to work with. This size is typically used in simple forms or when a select input is placed by itself on a page.
+
+{% contentPageImage {
+    src:"../../../assets/img/components/select-input/sizes-height-large.svg",
+    alt: "Large-sized select input.",
+    width: 256
+} %}
+
+---
+
+## Content
+
+All content should use sentence case.
+
+### Select option
+
+The recommended text is “Selected...”, but it can be modified if necessary with contextual changes.
+
+{% contentLayout %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/select-input/content-select-option-1.svg",
+      width: 266,
+      alt: "Select input showing dropdown with the first option preselected."
+    } %}
+  {% endcontentItem %}
+  {% contentItem %}
+    {% contentPageImage {
+      src: "../../../assets/img/components/select-input/content-select-option-2.svg",
+      width: 266,
+      alt: "Select input showing dropdown with the first option preselected."
+    } %}
+  {% endcontentItem %}
+{% endcontentLayout %}
```

Comment:
Not sure if I'm missing something, but both of these images display the same thing atm. Should one say `Selected...?`

![Screenshot 2025-04-07 at 16 01 19](https://github.com/user-attachments/assets/1acdf72a-5b2c-4e0a-8a18-c0c5b5e55791)

Reply:
updated